app-misc/openrgb-plugin-effects/openrgb-plugin-effects-0.9-r1.ebuild

```diff
@@ -23,6 +23,7 @@ KEYWORDS="amd64"
 
 RDEPEND="
 	>=app-misc/openrgb-0.9:=[-qt6(-)]
```

Comment:
It looks like this USEdep could also be cleaned up, now that the transition will not involve an IUSE qt6 (introduced in abcd889ece12a41ed5f8f729900a3125c80c1555 for transition) :)